packages/js/product-editor/src/components/section/edit.tsx

```diff
@@ -2,8 +2,18 @@
  * External dependencies
  */
 import { createElement } from '@wordpress/element';
-import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
 import type { BlockEditProps } from '@wordpress/blocks';
+import { InnerBlocks, useBlockProps } from '@wordpress/block-editor';
+import { sanitize } from 'dompurify';
+
+const ALLOWED_TAGS = [ 'a', 'b', 'em', 'i', 'strong', 'p', 'br' ];
+const ALLOWED_ATTR = [ 'target', 'href', 'rel', 'name', 'download' ];
+
+function sanitizeHTML( html: string ) {
```

Comment:
I think we can move this helper function to its own folder so we can reuse it in other places.

Reply:
Good idea!  Moved in https://github.com/woocommerce/woocommerce/pull/37455/commits/599b658d12dc3b8feafc45ef7accca148c94cd0d